service-proxy.js

```diff
@@ -453,17 +453,18 @@ function handleRequest(req, buildRes) {
         serviceChannel = self.createServiceChannel(nextService);
     }
 
-    if (serviceChannel.handler.circuits &&
-        (req.serviceName !== 'hyperbahn' && (
-            req.endpoint !== 'ad' || req.endpoint !== 'relay-ad'
-        ))
-    ) {
+    if (serviceChannel.handler.circuits) {
```

Comment:
this logic should be duplicated for lazy as well.

The reason the hyperbahn gaurd only happens in eager path is because ad() is an eager endpoint handler...